state/state.go

```diff
@@ -2666,3 +2673,12 @@ func TagFromDocID(docID string) names.Tag {
 		return nil
 	}
 }
+
+// RandomPrefix returns a random string.
+func RandomPrefix() (string, error) {
```

Comment:
This doesn't really belong in state - it's an artefact of k8s storage.
I think we can move it to `storage/kubernetes.go`